ngraph/test/frontend/paddlepaddle/test_models/gen_scripts/generate_slice.py

```diff
@@ -28,6 +29,75 @@ def slice(name : str, x, axes : list, start : list, end : list):
 
     return outs[0]
 
+
+'''
+export dyn model, long with input and output for reference.
+'''
+def exportModel(name, dyn_func, inputspecs:list, data):
+    model_dir = os.path.join(sys.argv[1], name)
+    save_path = '{}/{}'.format(model_dir, name)
+    pdpd.jit.save(dyn_func, save_path, inputspecs)
+
+    model = pdpd.jit.load(save_path)
+    result = model(data)
+    print(result.numpy().shape)
```

Comment:
only used for debug?

Reply:
exported as a helper function to export dyn graph of paddle.